source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -282,6 +328,20 @@ def _convert_direction(direction):
 
         return resp
 
+    def _get_language(self, language):
+        _language = _convert_here_language(language)
+        if _language not in Languages:
```

Comment:
i think the test is not mandatory... since you raise already an exception in convert_here_language